data/data-pipeline/data_pipeline/etl/base.py

```diff
@@ -77,8 +91,51 @@ class ExtractTransformLoad:
     #  periods. https://github.com/usds/justice40-tool/issues/964
     EXPECTED_MAX_CENSUS_TRACTS: int = 74160
 
+    # We use ourput_df as the final dataframe to use to write to the CSV
+    # It is used on the "load" base class method
     output_df: pd.DataFrame = None
 
+    @classmethod
+    def yaml_config_load(cls) -> dict:
+        # check if the class instance has score YAML definitions
+        datasets_config = load_yaml_dict_from_file(
+            cls.DATASET_CONFIG / "datasets.yml",
+            DatasetsConfig,
+        )
+
+        # get the config for this dataset
+        try:
+            dataset_config = next(
+                item
+                for item in datasets_config.get("datasets")
+                if item["module_name"] == cls.NAME
+            )
+        except StopIteration:
+            # Note: it'd be nice to log the name of the dataframe, but that's not accessible in this scope.
+            logger.error(
+                f"Exception encountered while extracting dataset config for dataset {cls.NAME}"
+            )
+            sys.exit()
+
+        # set some of the basic fields
+        cls.LAST_UPDATED_YEAR = dataset_config["last_updated_year"]
+        cls.SOURCE_URL = dataset_config["source_url"]
```

Comment:
couldn't we just pass a list?

Reply:
This is a bit more complicated, because if we pass a list of source urls, we also a corresponding list of extracted file names. Since this is not saving tremendous amounts of code, I can remove both properties and let the ETL handle manually.

Reply:
I have removed `source_url` and the input file name,